### DISCUSSION TITLE
New Powerwall Errors this Morning, Power Loss during TOU transition.

### DISCUSSION THREAD

**Discussion by s0schen** [jasonacox/Powerwall-Dashboard#238] [Q&A]
Anyone seen these errors:  SYNC DoCloseArguments,   FIFO_Overflow?

House lost power briefly and we switched over to grid when we were supposed to be on battery and solar for a bit.

![280E4561-7407-453A-A5A2-FE9A97A2A053](https://user-images.githubusercontent.com/4704443/230778886-abe30140-6ec5-4fe7-8cfa-14667407e568.jpeg)

**Comment by jasonacox** [comment]
Hi @s0schen - I saw SYNC_a046_DoCloseArguments during a prior Firmware version (22.26.4) and it would toggle off when I would manually take my system "Off Grid" (offset by SYNC_a038_DoOpenArguments).  However, it cleared with the next Firmware upgrade (22.36.6).  I haven't seen the THC_w061_CAN_TX_FIFO_Overflow.

**Reply by s0schen** [reply]
Thanks.    It did happen to be charging the car at the last minute before night rate ran out at 9a so we had a 13 kw load on the system.   When it happened power came back after 1 second and seemed to be coming off grid.   Tesla app showed no power coming off the panels even though the sun was out.    No data was being received by pypowerwall to the Dashboard for about 10 minutes.    Otherwise the the system has been rock solid since PTO Dec 22 and we've never even noticed when the grid actually did go offline.

**Reply by jasonacox** [reply]
Very curious. Can you post the main "Live Monitoring" Energy Usage graph during that time? 

> No data was being received by pypowerwall to the Dashboard 

That's odd! Was it like the gateway rebooted or did your network or WiFi go down during the outage?

**Reply by s0schen** [reply]
I think some component of the tesla rebooted.   Wifi was lost but I assume the Tesla app still communicates via cellular and you can see the solar drop out and the powerwall stops charging during the glitch.

![87DB23EA-F2DA-43A6-A120-617056AFC3CE](https://user-images.githubusercontent.com/4704443/230896423-71a12397-4e30-44bb-a181-c28cfd8dcf35.jpeg)
![E6B5F368-E73E-40BE-9750-1C3E5997A1ED_1_105_c](https://user-images.githubusercontent.com/4704443/230896455-b1dc1038-7d85-499b-bd1c-732344b710e2.jpeg)
![CE876886-D814-4FD6-B712-D35A3BDBC3C0_1_105_c](https://user-images.githubusercontent.com/4704443/230896478-a9988f23-e81e-4ef8-8a6e-2ad6796ba989.jpeg)

**Reply by jasonacox** [reply]
Nice! Thanks @s0schen .  The fact that you also lost weather data during that timeframe indicates that your dashboard host either lost power itself or it could no longer connect to the internet (weather is an internet call, not a call to the Powerwall).  In any case, you can see the grid usage was high (EV charging) until the event and went to zero.
